.travis.yml

```diff
@@ -91,4 +94,6 @@ matrix:
         - os: linux
           compiler: gcc
           env: COMPILER_VERSION=4.8 CXXSTD=c++11 DEBUG=1
-
+        - os: linux
+          compiler: gcc
+          env: COMPILER_VERSION=6 CXXSTD=c++14 DEBUG=1
```

Comment:
Any reason to just have the debug build?

Reply:
No, that's a mistake on my part. I intended to add only a non-debug build as I don't think it's a good use of Travis resources to have more than one debug build. Just messed it up. I've force-pushed a fix.